pkg/provider/acme/challenge_tls.go

```diff
@@ -68,6 +68,12 @@ func (c *ChallengeTLSALPN) Present(domain, _, keyAuth string) error {
 	case t := <-timer.C:
 		timer.Stop()
 		close(c.chans[string(certPEMBlock)])
+
+		err = c.CleanUp(domain, "", keyAuth)
+		if err != nil {
+			logger.Errorf("failed to clean up: %v", err)
```

Comment:
Even if the `CleanUp` method never returns an error, maybe the following log message is better:

```suggestion
			logger.Errorf("Failed to clean up TLS challenge: %v", err)
```